### ISSUE TITLE
NTR: Oligoclonal bands of CSF protein (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8446]
New term request Oligoclonal bands of CSF protein (segal_070124171538): Polyclonal gammopathy; cerebrospinal fluid; typically IgG but can be IgM or IgA  UMLS: 3887781

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000397
- label: CSF oligoclonal immunoglobulin G bands
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: PMID:35277112
- def: Oligoclonal immunoglobulin G (IgG) bands (OCBs) are a useful diagnostic tool to detect a central humoral response. In particular, cerebrospinal fluid (CSF)-restricted OCBs represent a hallmark of multiple sclerosis (MS).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8446
- synonyms:
